.github/workflows/changelog-update.yml

```diff
@@ -0,0 +1,47 @@
+name: Update changelog
+
+on:
+  pull_request_review:
+    types: [ submitted ]
```

Comment:
This will trigger every time a PR review is submitted. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-when-a-pull-request-is-approved

Is the `merged` option what we are actually looking for? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-when-a-pull-request-merges